packages/admin/admin/src/common/FieldSet.tsx

```diff
@@ -111,14 +115,15 @@ const Summary = createComponentSlot(MuiAccordionSummary)<FieldSetClassKey>({
     componentName: "FieldSet",
     slotName: "summary",
 })(
-    ({ theme }) => css`
+    ({ theme, ownerState }) => css`
```

Comment:
The `ownerState` is no longer used here

```suggestion
    ({ theme }) => css`
```